lib/abe_client/subscriber.rb

```diff
@@ -10,15 +10,15 @@ class Subscriber
     GRAPHQL
 
     def self.add(blog, email_address, first_name = nil, last_name = nil, identity_uid = nil)
-      result = Client.query(AddSubscriberMutation, {
+      result = Client.query(AddSubscriberMutation,
         :variables => {
```

Comment:
Optional: if you want you can update the symbol hash syntax, eg `variables: {`

Reply:
yeah I considered it, but I thought it was better to stay consistent with all the existing code. And I didn't feel it would be right to go around changing everything 😅